tests/Rector/Architecture/DependencyInjection/AnnotatedPropertyInjectToConstructorInjectionRector/Correct/correct2.php.inc

```diff
@@ -1,19 +1,19 @@
 <?php declare (strict_types=1);
 
-class ClassWithInjects
+class ClassWithInjects2
 {
     /**
      * @var stdClass
+     * @autowired
      */
-    private $property;
+    public $property;
 
     /**
      * @var DateTimeInterface
-     * @inject
      */
-    public $otherProperty;
-    public function __construct(\stdClass $property)
+    private $otherProperty;
+    public function __construct(\DateTimeInterface $otherProperty)
     {
-        $this->property = $property;
+        $this->otherProperty = $otherProperty;
     }
 }
```

Comment:
Why this needs a change?

Reply:
yeah, sorry, should have been split to a separate commit.

this was an old test file that wasn't being used any more. it tested a the `autowired ` annotation.
https://github.com/rectorphp/rector/commit/01652b1f9e4f6e2594400238dbc5995662a74131
I've corrected the expected result because the current test works on the `inject` annotation.

Reply:
No need for commit, but rather completely new file.

Reply:
For clarity, that test file was not being used.
The test was deleted by the `php.inc` files were not.
https://github.com/rectorphp/rector/commit/8dcc357104af24ef4810045cfee53756b9748778

Reply:
Ah, I see. My bad :)